test/e2e/support/utils/get-url.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Node dependencies
+ */
+import { join } from 'path';
+
+const {
+	WP_BASE_URL = 'http://localhost:8889',
+} = process.env;
```

Comment:
This is another const which could be grouped with other similar variables based on `process.env`.

Reply:
Thx, updated: https://github.com/WordPress/gutenberg/pull/11701/commits/406ff7ae3fed7ccc0e323116f7091aef17348543